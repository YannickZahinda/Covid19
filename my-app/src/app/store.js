import { configureStore } from "@reduxjs/toolkit";
import casesReducer from '../Redux/home';
import detailReducer from '../Redux/info';
import searchReducer from '../Redux/search'

const store = configureStore({
    reducer: {
        covid: casesReducer,
        details: detailReducer,
        search: searchReducer
    },
});

export default store;