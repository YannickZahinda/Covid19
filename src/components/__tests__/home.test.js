import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../app/store';
import Home from '../home/home';

describe('missions Component test', () => {
  it('Renders Mission Row', () => {
    const app = renderer
      .create(
        <Provider store={store}>
          <Home />
        </Provider>,
      )
      .toJSON();
    expect(app).toMatchSnapshot();
  });
});