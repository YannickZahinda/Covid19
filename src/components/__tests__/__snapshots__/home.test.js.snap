// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`missions Component test Renders Mission Row 1`] = `
<div
  className="all-countries"
>
  <div>
    <div
      className="searbar"
    >
      <form>
        <input
          className="input-search"
          onChange={[Function]}
          placeholder="search a country..."
          type="search"
          value=""
        />
      </form>
    </div>
  </div>
  <div
    className="country-container"
  />
</div>
`;
